packages/fungible-token/bridge-fungible-token/src/cast.sw

```diff
@@ -0,0 +1,82 @@
+library;
+
+use std::u256::U256;
+use std::constants::ZERO_B256;
+
+impl From<b256> for U256 {
+    fn from(value: b256) -> U256 {
+        let (word1, word2, word3, word4) = asm(r1: value) { r1: (u64, u64, u64, u64) };
+        let result = U256::from((word1, word2, word3, word4));
+
+        result
+    }
+
+    fn into(self) -> b256 {
+        let result: b256 = ZERO_B256;
```

Comment:
```suggestion
        let result = b256::min();
```

No need to annotate the type here.
You could use either type afaik.

I'd also remove the import for the constant if you accept these changes which use a different import